frontend/public/locales/en/new-store-business.json

```diff
@@ -3,7 +3,7 @@
   "address": "Address of your store",
   "city": "City",
   "zip": "ZIP",
-  "vatNo": "VAT identification number",
+  "vatNo": "VAT Number  *This is required to verify your business before your store will be visible to customers.",
```

Comment:
Have you tested this? This won't even be shown completely on phones, it is the text that is shown in the input field.

Reply:
It'd better be in `text`